scripts/cob-command

```diff
@@ -162,6 +165,14 @@ stop_session(){
     fi
     su $USER -l -c "ssh $PC 'tmux kill-session -t $arg_session_stop'"
   fi
+
+  if ! $lock_slack; then
+    su $USER -l -c "ssh $PC 'tmux send -t $arg_session_stop C-c'"
+    if [ ! -z "$PID" ]; then
+      while su $standard_user -l -c "ssh $PC stat /proc/$PID \> /dev/null 2\>\&1 "; do echo "still waiting for $arg_session_stop to stop (PID $PID)" && sleep 1; done
+    fi
+    su $USER -l -c "ssh $PC 'tmux kill-session -t $arg_session_stop'"
+  fi
```

Comment:
what does this block do exactly?
why do you need this for slack and not for core?